project/Build.scala

```diff
@@ -12,7 +12,7 @@ object DottyBuild extends Build {
   val baseVersion = "0.1"
   val isNightly = sys.env.get("NIGHTLYBUILD") == Some("yes")
 
-  val jenkinsMemLimit = List("-Xmx1300m")
+  val jenkinsMemLimit = List("-Xmx1500m")
```

Comment:
Previously dotty was not compiled properly by partest. It was only able to compile top-level files (since the compiler itself was on the class path).

It never recursed obeying `-deep`.